applications/reference-data/src/main/resources/rdf/api-status-skos.ttl

```diff
@@ -0,0 +1,31 @@
+@prefix at: <http://publications.europa.eu/ontology/authority/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@base <http://fellesdatakatalog.brreg.no/reference-data/codes/apistastus/> .
+
+<api-statuses>
+    a skos:ConceptScheme ;
+    skos:prefLabel "Status på API-et"@nb .
+
+<production>
+    a skos:Concept ;
+    at:authority-code "production";
+    skos:inScheme   <api-statuses> ;
+    skos:prefLabel "I produksjon"@nb .
+
+<nonproduction>
+    a skos:Concept ;
+    skos:inScheme   <api-statuses> ;
+    at:authority-code "non-production";
+    skos:prefLabel "Ikke produsjonsatt"@nb .
+
+<under-deprecation>
```

Comment:
will-be-deprecated eller to-be-deprecated er bedre engelsk.

Reply:
takk!, jeg skal bytte til de koder jeg anbefaler til vi får svar fra innføringsteamet

Reply:
kodene er oppdatert, venter oversetningne til engelsk og nynorsk